example/take_screenshot.js

```diff
@@ -0,0 +1,17 @@
+const bin = require('ffmpeg-binaries').ffmpegPath();
+const cp = require('child_process');
+const ytdl = require('..');
+
+const takeScreenshot = (url, outFile, position) => new Promise((resolve, reject) => {
+  ytdl.getInfo(url).then(info => {
+    let format = info.formats[0];
+    cp.exec(`"${bin}" -i "${format.url}"${!format.live && position ? ` -ss ${position}` : ''} -vframes 1 -an -y ${outFile}`, (error, stdout, stderr) => {
+      if(error) return reject({error, stdout, stderr});
```

Comment:
Space after `if`, after `{`, and before `}` to keep style consistent across the repo